activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -500,7 +500,7 @@ def extensions
       def enum_types
         query = <<~SQL
           SELECT
-            type.typname AS name,
+            quote_ident(type.typname) AS name,
```

Comment:
[`quote_ident`](https://pgpedia.info/q/quote_ident.html) was added in PG 7.2 (April 2001), so I assume version support isn't a concern 😅